application/src/main/java/org/togetherjava/tjbot/commands/moderation/report/package-info.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * This packages offers all the functionality for messaging moderators trough report see
+ * {@link org.togetherjava.tjbot.commands.moderation.report.ReportCommand} as main entry point.
+ */
+@MethodsReturnNonnullByDefault
+@ParametersAreNonnullByDefault
+package org.togetherjava.tjbot.commands.moderation.report;
```

Comment:
y have a package for a single file, this should be in moderation package, no?

Reply:
Its the same reason for Modmail and Blacklisted. And it is these ones are more specific, and could later be more broadened (probably)

Reply:
agreed. single file stuff shouldnt be in its own package. if the package is already too crowded with other stuff the package itself should be reorganized

Reply:
Okay I will move it to package above

Reply:
Done.